include/swift/AST/Decl.h

```diff
@@ -5948,6 +5948,10 @@ class AbstractFunctionDecl : public GenericContext, public ValueDecl {
   /// Returns true if the function is an @asyncHandler.
   bool isAsyncHandler() const;
 
+  /// Returns true if the function if the signature matches the form of an
```

Comment:
It the statement "if the function if the signature matches..." is correct? Or it should be something like "if the function signature matches..."?

Reply:
"If the function signature matches" is correct; will fix in a follow-up.